### ISSUE TITLE
[Bug] scoop update mysql mysql: 8.3.0 -> 8.4.0

### ISSUE THREAD

**Issue by dong-lufei** [ScoopInstaller/Main#5777]
```sh
# dong @ lufei in E:\test [22:24:11]
$ scoop update mysql
mysql: 8.3.0 -> 8.4.0
Updating one outdated app:
Updating 'mysql' (8.3.0 -> 8.4.0)
Downloading new version
Loading mysql-8.4.0-winx64.zip from cache.
Checking hash of mysql-8.4.0-winx64.zip ... ok.
Uninstalling 'mysql' (8.3.0)
Removing shim 'ibd2sdi.shim'.
Removing shim 'ibd2sdi.exe'.
Removing shim 'innochecksum.shim'.
Unlinking D:\scoop_user_apps\apps\mysql\current
Installing 'mysql' (8.4.0) [64bit] from main bucket
Loading mysql-8.4.0-winx64.zip from cache.
Extracting mysql-8.4.0-winx64.zip ... done.
Running pre_install script...
Linking D:\scoop_user_apps\apps\mysql\current => D:\scoop_user_apps\apps\mysql\8.4.0
Creating shim for 'ibd2sdi'.
Creating shim for 'innochecksum'.
WARN  Overwriting shim ('innochecksum.exe' -> 'innochecksum.exe') installed from mariadb
Creating shim for 'lz4_decompress'.
Can't shim 'bin\lz4_decompress.exe': File doesn't exist.
# dong @ lufei in E:\test [22:24:24] C:1
```

### System details

**Windows version:** [e.g. 7, 8, 10, 11]
11
**OS architecture:** [e.g. 32bit, 64bit, arm64]

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
//# Your configuration here
```

**Comment by flybyray** [comment]
Needs to be fixed

```
Removed the deprecated mysqlpump utility along with its associated lz4_decompress and zlib_decompress helper utilities. Instead, use mysqldump or [MySQL Shell's dump utilities](https://dev.mysql.com/doc/mysql-shell/8.4/en/mysql-shell-utilities-dump-instance-schema.html). (WL #16096)
```
ref: https://dev.mysql.com/doc/relnotes/mysql/8.4/en/news-8-4-0.html

**Comment by dong-lufei** [comment]
I cleared the cache with the command scoop cache rm * and re-upgrading mysql still failed. Does this version not upgrade this way, can only uninstall and reinstall version 8.4.0
```
# dong @ lufei in E:\test [17:10:16]
$ scoop update mysql
mysql: 8.3.0 -> 8.4.0
Updating one outdated app:
Updating 'mysql' (8.3.0 -> 8.4.0)
Downloading new version
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 51a259|OK  |   5.6MiB/s|D:/scoop_user_apps/cache/mysql#8.4.0#https_dev.mysql.com_get_Downloads_MySQL-8.4_mysql-8.4.0-winx64.zip
Download: Status Legend:
Download: (OK):download completed.
Checking hash of mysql-8.4.0-winx64.zip ... ok.
Uninstalling 'mysql' (8.3.0)
Removing shim 'ibd2sdi.shim'.
Removing shim 'ibd2sdi.exe'.
Removing shim 'innochecksum.shim'.
Unlinking D:\scoop_user_apps\apps\mysql\current
Installing 'mysql' (8.4.0) [64bit] from main bucket
Loading mysql-8.4.0-winx64.zip from cache.
Extracting mysql-8.4.0-winx64.zip ... done.
Running pre_install script...
Linking D:\scoop_user_apps\apps\mysql\current => D:\scoop_user_apps\apps\mysql\8.4.0
Creating shim for 'ibd2sdi'.
Creating shim for 'innochecksum'.
WARN  Overwriting shim ('innochecksum.exe' -> 'innochecksum.exe') installed from mariadb
Creating shim for 'lz4_decompress'.
Can't shim 'bin\lz4_decompress.exe': File doesn't exist.
# dong @ lufei in E:\test [17:11:21] C:1
```

**Comment by benjivm** [comment]
While waiting for the PR for this issue to be merged I downgraded to 8.3.0, which requires skipping hash check:

```bash
scoop install mysql@8.3.0 -s
```
